fife/__main__.py

```diff
@@ -34,206 +34,190 @@ def main():
         raise NotImplementedError(
             "Forecasting future state from the command line is not yet supported. Try LGBStateModeler from the FIFE Python package."
         )
+
     utils.make_results_reproducible(config["SEED"])
     utils.redirect_output_to_log(path=config["RESULTS_PATH"])
     utils.print_copyright()
     utils.print_config(config)
-    data = read_data(config)
+    df = read_data(config)
     print(f"I/O setup time: {time() - checkpoint_time} seconds")
 
-    checkpoint_time = time()
-    data_processor = processors.PanelDataProcessor(config, data)
-    data_processor.build_processed_data()
-    utils.save_intermediate_data(
-        data_processor.data,
-        "Processed_Data",
-        file_format="pickle",
-        path=config["RESULTS_PATH"],
-    )
-    print(f"Data processing time: {time() - checkpoint_time} seconds")
-
-    checkpoint_time = time()
-    utils.ensure_folder_existence(f'{config["RESULTS_PATH"]}/Intermediate/Models')
-    test_intervals = config.get("TEST_INTERVALS", config.get("TEST_PERIODS", 0) - 1)
-    if config.get("TREE_MODELS"):
-        modeler_class = lgb_modelers.LGBSurvivalModeler
-    elif config.get("PROPORTIONAL_HAZARDS"):
-        modeler_class = tf_modelers.ProportionalHazardsModeler
-    else:
-        modeler_class = tf_modelers.FeedforwardNeuralNetworkModeler
-    modeler = modeler_class(config=config, data=data_processor.data)
-    modeler.n_intervals = (
-        test_intervals if test_intervals > 0 else modeler.set_n_intervals()
-    )
-    if config.get("HYPER_TRIALS", 0) > 0:
-        params = modeler.hyperoptimize(config["HYPER_TRIALS"])
-    else:
-        params = None
-    modeler.build_model(n_intervals=modeler.n_intervals, params=params)
-    modeler.save_model(path=f"{config['RESULTS_PATH']}/Intermediate/Models/")
-    print(f"Model training time: {time() - checkpoint_time} seconds")
-
-    checkpoint_time = time()
-    if test_intervals > 0:
-
-        # Save metrics
-        evaluation_subset = modeler.data["_period"] == (
-            modeler.data["_period"].max() - test_intervals
-        )
-        utils.save_output_table(
-            modeler.evaluate(evaluation_subset),
-            "Metrics",
-            path=config["RESULTS_PATH"],
+    by_feature = config.get("BY_FEATURE", None)
+    if by_feature == "None":
+        by_feature = None
+    if by_feature != None and by_feature not in df.columns:
```

Comment:
Please use `is not` instead of `!=` when comparing an object with `None`. See [this post](https://stackoverflow.com/questions/2209755/python-operation-vs-is-not/2209781).